python/langsmith/evaluation/_arunner.py

```diff
@@ -329,6 +329,7 @@ async def aevaluate_existing(
         max_concurrency=max_concurrency,
         client=client,
         blocking=blocking,
+        experiment=project,
```

Comment:
Not releated to incrementality - just realized in my UT we were doing an extra unnecessary call